server/GraknServer.java

```diff
@@ -153,74 +127,79 @@ private void configureTracing() {
         }
     }
 
-    private static void printASCIILogo() throws IOException {
-        if (ASCII_LOGO_FILE.exists()) {
-            System.out.println("\n" + new String(Files.readAllBytes(ASCII_LOGO_FILE.toPath()), StandardCharsets.UTF_8));
-        }
+    protected Server rpcServer() {
+        assert Executors.isInitialised();
+
+        graknService = new GraknService(grakn);
```

Comment:
The service classes are now instantiated here rather than the constructor, for consistency with how we construct it in Grakn Cluster. In Grakn Cluster, we will instantiate an additional service for clustering.